Dmitriy_Vysotskyy2.0/Steps/DeleteFromCartSteps.cs

```diff
@@ -0,0 +1,41 @@
+using Dmitriy_Vysotskyy2._0.Builders;
+using Dmitriy_Vysotskyy2._0.Common;
+using FluentAssertions;
+using TechTalk.SpecFlow;
+
+namespace Dmitriy_Vysotskyy2._0.Steps;
+
+[Binding]
+public class DeleteFromCartSteps : FeatureHelper
+{
+    [Given(@"an existing user has logged in")]
+    public void GivenAnExistingUserHasLoggedIn()
+    {
+        var user = new UserModelBuilder()
+            .SetLogin("charles")
+            .SetPassword("charles")
+            .Build();
+        _indexPage = _homePage.LogIn(user.Login, user.Password);
+    }
+
+    [Given(@"user added ""(.*)"" to cart")]
```

Comment:
You have the same step in this class and in PlaceOrderSteps. Please create common class called OrderSteps or smth like this and put common code there

Reply:
and review all steps and put duplicates in common places